Frontend/student-hub/src/pages/MentorshipPage.js

```diff
@@ -68,10 +92,11 @@ const delayTime = 60000;
         }
 
         const mentorshipData = await response.json();
+        const mentors = mentorshipData.formattedUser.filter((user) => user.mentorship !== 'Mentee');
```

Comment:
probably more accurate to filter exactly on 'Mentor' instead of the negation here. That way if there are future mentorship roles you don't get a bug here